frontend/document/credocument.lua

```diff
@@ -1509,6 +1509,7 @@ function CreDocument:register(registry)
     registry:addProvider("chm", "application/vnd.ms-htmlhelp", self, 90)
     registry:addProvider("doc", "application/msword", self, 90)
     registry:addProvider("docx", "application/vnd.openxmlformats-officedocument.wordprocessingml.document", self, 90)
+    registry:addProvider("docm", "application/vnd.ms-word.document.macroEnabled.12", self, 90)
```

Comment:
lol

Reply:
One could say that's a quality... memetype. *kaching*!